src/Aquifer.Jobs/Subscribers/EmailSubscriber.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Aquifer.Common.Jobs;
+using Aquifer.Common.Services;
+using Aquifer.Jobs.Configuration;
+using Aquifer.Jobs.Services;
+using Azure.Storage.Queues.Models;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Aquifer.Jobs.Subscribers;
+
+public sealed class EmailSubscriber(
+    [FromKeyedServices(nameof(SendGridEmailService))] IEmailService _emailService,
+    IOptions<ConfigurationOptions> _configurationOptions,
+    ILogger<EmailSubscriber> _logger)
+{
+    [Function(nameof(SendEmail))]
+    public async Task SendEmail(
+        [QueueTrigger(Queues.SendEmail)]
+        QueueMessage message,
+        CancellationToken ct)
+    {
+        var email = message.Deserialize<Email, EmailSubscriber>(_logger);
+
+        try
+        {
+            await _emailService.SendEmailAsync(
+                email with { Subject = $"{_configurationOptions.Value.Email.SubjectPrefix}{email.Subject}" },
+                ct);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "Unable to send email: \"{Email}\"", message.MessageText);
```

Comment:
Previously failed email sends were logging at error in the telemetry client: https://github.com/BiblioNexusStudio/aquifer-server/blob/ecd3030ae68e27cd9f9cced63886218f60dc3818/src/Aquifer.Jobs/SendNewContentEmails.cs#L86.  Is that necessary here?  It's not clear to me when using a telemetry client is necessary and what we get for free with standard exception and `ILogger<T>`.